assets/questions/isLeapYear.js

```diff
@@ -0,0 +1,196 @@
+        'leap year, and False otherwise. For example:'
+      ].join(''),
+      type: 'text'
+    }, {
+      content: 'Input: 2018\nOutput: False',
+      type: 'code'
+    }],
+    prerequisiteSkills: [],
+    acquiredSkills: ['Conditionals'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'isLeapYear',
+    languageSpecificTips: {
+      python: []
+    },
+    testSuites: [{
+      id: 'NOT_DIVISIBLE_BY_FOUR',
+      humanReadableName: 'the not-divisible-by-4 case',
+      testCases: [{
+        input: 1903,
+        allowedOutputs: [false]
+      }, {
+        input: 1,
+        allowedOutputs: [false]
+      }, {
+        input: 1905,
+        allowedOutputs: [false]
+      }, {
+        input: 702,
+        allowedOutputs: [false]
+      }]
+    }, {
+      id: 'DIVISIBLE_BY_FOUR_BUT_NOT_HUNDRED',
+      humanReadableName: 'the divisible-by-4-but-not-100 case',
+      testCases: [{
+        input: 1904,
+        allowedOutputs: [true]
+      }, {
+        input: 1976,
+        allowedOutputs: [true]
+      }, {
+        input: 2004,
+        allowedOutputs: [true]
+      }, {
+        input: 2016,
+        allowedOutputs: [true]
+      }]
+    }, {
+      id: 'DIVISIBLE_BY_HUNDRED_BUT_NOT_FOUR_HUNDRED',
+      humanReadableName: 'the divisible-by-100-but-not-400 case',
+      testCases: [{
+        input: 1900,
+        allowedOutputs: [false]
+      }, {
+        input: 1700,
+        allowedOutputs: [false]
+      }, {
+        input: 2100,
+        allowedOutputs: [false]
+      }, {
+        input: 3400,
+        allowedOutputs: [false]
+      }]
+    }, {
+      id: 'DIVISIBLE_BY_FOUR_HUNDRED',
+      humanReadableName: 'the divisible-by-400 case',
+      testCases: [{
+        input: 2000,
+        allowedOutputs: [true]
+      }, {
+        input: 1600,
+        allowedOutputs: [true]
+      }, {
+        input: 400,
+        allowedOutputs: [true]
+      }, {
+        input: 1200,
+        allowedOutputs: [true]
+      }]
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.returnNone',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          'It looks like your program is not returning anything. Did you ',
+          'forget to include a "return" statement?'
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.returnTrue',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          "It looks like you're consistently returning True for all test ",
+          "cases. This implies that every year is a leap year, but that's ",
+          "not true..."
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.returnFalse',
+      ignoredTestSuiteIds: [],
+      messages: [
+        [
+          "It looks like you're consistently returning False for all test ",
+          "cases. This implies that there are no leap years, but that's not ",
```

Comment:
This implies there are no leap years, but that is not true.

Reply:
Done. (Just for my edification, what was the primary issue with the original here?)

Reply:
It was the ellipses, yeah. I prefer a more authoritative tone when we're trying to like, tell them what's wrong. Trailing off seems like we're unsure. Or that's the impression I get.